_data/toc.yaml

```diff
@@ -1439,12 +1439,12 @@ manuals:
   - sectiontitle: Install Compose
     section:
```

Comment:
An alternative here could be to move the `/compose/install/` to the top level. That way getting to the `install` page would be a "one click", instead of two (click to expand the sub-menu, then click to go to `install`). The section could then be renamed to (e.g. "Installation per platform" - similar to "Installation per distro" in the engine manual). Most users (users with Desktop installed) would never have to look inside that section.

### Install page at the top level

<img width="600" alt="Screenshot 2022-08-29 at 17 42 59" src="https://user-images.githubusercontent.com/1804568/187240540-7dee5f09-d267-421e-8d43-e76d5f5ca80c.png">

### .. and for those that really need the other install methods..

<img width="600" alt="Screenshot 2022-08-29 at 17 43 07" src="https://user-images.githubusercontent.com/1804568/187240554-0c5320a3-42fb-4fb1-afc3-6b012e98d883.png">

Note: For consideration; I also renamed the "Overview of Docker Compose" and "Install Docker Compose" in the TOC to just "overview" and "install".  That approach would provide a consistent navigation in each of the manual ("overview", "install", ...)

<img width="273" alt="Screenshot 2022-08-29 at 17 46 18" src="https://user-images.githubusercontent.com/1804568/187241351-0480cb3f-a4ed-4706-98d0-69a0a558a7a3.png">
<img width="265" alt="Screenshot 2022-08-29 at 17 46 26" src="https://user-images.githubusercontent.com/1804568/187241358-bde0c0e1-6494-4a9f-907a-00e6bf92eb3b.png">
<img width="253" alt="Screenshot 2022-08-29 at 17 46 30" src="https://user-images.githubusercontent.com/1804568/187241360-766af987-309d-424a-a5e3-03f7c9a946dd.png">

Reply:
Good point about using "Overview" to make Compose subnav consistent with other sections! 

But, for this PR I don't think we should block on this suggestion of a separate "Install" link and another "Installation per platform" section.

| Dora's PR | Suggested Change |
| --- | --- |
| <img width="250" alt="image" src="https://user-images.githubusercontent.com/20349/187323819-7b1a5815-2c99-44b1-9b31-032a38dc8c21.png"> | <img width="250" alt="image" src="https://user-images.githubusercontent.com/20349/187323865-e00a9d12-1c5a-4b00-9d05-f2544121e286.png"> | 


"Per platform" suggests you'll see a list of links by platform. But actually the content available is a kind of grab-bag of install-related topics: the Linux plugin install, "other scenarios" that group Linux standalone and Windows Server, and Uninstall. So, it doesn't really seem like an accurate group description.

My suggestion is that we stick with what Dora has in this PR, with everything under "Install Compose." We do have a similar precedent with the "Install Docker Desktop" info hierarchy:

| Dora's PR | Current Docs |
| --- | --- |
| <img width="250" alt="image" src="https://user-images.githubusercontent.com/20349/187323819-7b1a5815-2c99-44b1-9b31-032a38dc8c21.png"> | <img width="250" alt="image" src="https://user-images.githubusercontent.com/20349/187323417-922ac40b-9826-480c-9133-195ac339f715.png"> |


&nbsp;
&nbsp;
&nbsp;
That said…now I'm looking more closely at this, it does feel redundant to have:

```
Install Compose
  Install Docker Compose
  …
```

So how about this instead?

```
Install
  Install Docker Compose
  …
```

Reply:
> But actually the content available is a kind of grab-bag of install-related topics

Yes, it definitely is, and I was somewhat hoping we could keep that hidden for most users (who have Desktop installed). Had a bit of a hard time coming up with a section title for that ("Advanced install"? "Other installation methods"?). 

> That said…now I'm looking more closely at this, it does feel redundant to have:

Yes, that was the other reason I was looking at changing.

> My suggestion is that we stick with what Dora has in this PR

... after all of the above said; I'm ok with refining after this if we don't (yet?) want to change. That is; assuming it will be changes in the TOC only (restructuring the menu); where possible we should avoid changing URLs / renaming files (as moving them around may also cause penalties from search engines / ranking). So if we address those before merging (e.g. https://github.com/docker/docker.github.io/pull/15494#discussion_r957487060), then all should be fine.

Reply:
* Changed the title of Overview of Compose (...) to follow what we're doing in other doc sets. 
* I haven't changed the Install section title - we can do it later on if we want all three doc sets to have that title instead.
* Changed the index to "Install overview" - this way it is also clearer there are no actual instructions in this page.
___
* Some inconsistencies in docs are not about docs, so it will be hard to solve them with docs.
* Engine installation is more information-dense and distributed across platforms/distros and will also need some consolidation IMHO.
* Personally, I find it confusing to have an "Install" page and then a "Install per distro" section below it. These are not independent topics, and so, do not merit a separate item in the TOC and should be grouped together.